weather_mv/loader_pipeline/ee.py

```diff
@@ -478,33 +507,49 @@ def start_ingestion(self, asset_request: t.Dict) -> str:
         self.check_setup()
 
         try:
-            result = ee.data.createAsset(asset_request)
+            if self.ee_asset_type == 'IMAGE':
+                result = ee.data.createAsset(asset_request)
+            elif self.ee_asset_type == 'TABLE':
+                task_id = ee.data.newTaskId(1)[0]
+                result = ee.data.startTableIngestion(task_id, asset_request)
         except ee.EEException as e:
             logger.error(f"Failed to create asset '{asset_request['name']}' in earth engine: {e}")
             raise
 
         return result.get('id')
 
-    def process(self, tiff_data: TiffData) -> t.Iterator[str]:
-        """Uploads a tiff image into the earth engine."""
-        target_path = tiff_data.target_path
-        asset_name = os.path.join(self.ee_asset, tiff_data.name)
-        start_time = tiff_data.start_time
-        end_time = tiff_data.end_time
-        properties = tiff_data.properties
+    def process(self, asset_data: AssetData) -> t.Iterator[str]:
+        """Uploads an asset into the earth engine."""
+        target_path = asset_data.target_path
+        asset_name = os.path.join(self.ee_asset, asset_data.name)
+        start_time = asset_data.start_time
+        end_time = asset_data.end_time
+        properties = asset_data.properties
```

Comment:
Optional style nit: My personal preference is that extracting out these variables is unnecessary; you can just access them from `asset_data`.